test/hotspot/jtreg/runtime/verifier/CFLH/TestVerify.java

```diff
@@ -164,7 +163,8 @@ public static void main(String argv[]) throws Exception {
             } else {
                 // Load the class instrumented with CFLH for the VerifyError.
                 inst.addTransformer(new BadTransformer());
-                System.out.println("1 hour is " + Duration.ofHours(1).getSeconds() + " seconds");
+                Class<?> cls = Class.forName(CLASS_TO_BREAK);
+                System.out.println("class loaded" + cls);
```

Comment:
Isn't this println redundant, because the VerifyError is supposed to be thrown before? I think the old code uses `Durations.ofHours` to trigger verification and initialization of Duration.

Reply:
It is not reached since we throw the VerifyError at line 166.